frontend/src/components/grantaccess/form/UserNameInput.vue

```diff
@@ -28,20 +32,43 @@ const computedUserId = computed({
     },
 });
 
-const verifiedUserIdentity = ref<IdimProxyIdirInfo | null>(null);
+const cardMgs = ref('');
+const verifiedUserIdentity = ref<IdimProxyIdirInfo | IdimProxyBceidInfo | null>(
+    null
+);
 const verifyUserId = async () => {
-    if (props.domain == UserType.B) {
-        emit('setVerifyResult', true);
-        return;
+    try {
+        if (props.domain == UserType.B) {
+            verifiedUserIdentity.value = (
+                await AppActlApiService.idirBceidProxyApi.bceidSearch(
+                    computedUserId.value
+                )
+            ).data;
+        } else {
+            verifiedUserIdentity.value = (
+                await AppActlApiService.idirBceidProxyApi.idirSearch(
+                    computedUserId.value
+                )
+            ).data;
+        }
+    } catch (error: any) {
```

Comment:
this is a good catch, we missed error handling previously. Even now we catch the permission error and customize the error message, but for other type of errors, I didn't see the toast message popup, feel it's not got caught. But we can create a follow up ticket to handle the rest error cases, might need to wait for Olga's design cause it might not make sense to use toast here as well, might need to follow how we display errors for verify forest client numbers. So I think Jota if you can create a follow up ticket for handling other errors that should be okay for now. How you feel Ian @ianliuwk1019 ?

Reply:
Sorry, maybe I am not clear. 
What were the errors that need to be caught here other than the business rule that BCeID user can only very BCeID user within the same organization?
In the case of not directly related to the business rule here in this input, I would leave it for general handling (however, I seem to remember that was removed long time ago).
If it is for business rule validation, I feel it make sense if Olga wants to display in this page with some message; but if Olga designs to use toast message it can still be acceptable just need to be consistent with other pages..

However, the caught error: "**PERMISSION_REQUIRED_FOR_OPERATION**" seems a bit strange; the component should expect to catch some error related to (400, 409 or 422) in the component I think.

Reply:
yeah the general error is not triggering the toast error message currently, so Jota if you could create a follow up ticket for that, thanks. And for this pr, let's change to check both status code 409 and the detail code, thanks!!

Reply:
Sure, will do!